mapbox/dr-ui
# Remove IE 11 support

_This PR merges into #393._

This PR removes IE 11 support for Dr. UI by removing IE 11 specific polyfills and other support methods.

## How to test

I pushed this release to /playground. It's currently on staging. You could open staging /playground on various browsers, especially Edge, to make sure they load as expected.

## QA checklist




Open the test cases app locally on:
